account_payment_return_import_iso20022/wizard/payment_return_import.py

```diff
@@ -45,6 +46,12 @@ def _parse_single_document(self, data_file):
         """
         camt_parser = CamtParser()
         pain_parser = PainParser()
+        # If we are in test mode we only want this behavior when we pass that context
+        # key, that is, when we are running the tests of that module.
+        if config["test_enable"] and not self.env.context.get(
```

Comment:
Why is this needed?

Reply:
To avoid this error now that the tests are in post-install https://github.com/OCA/account-payment/actions/runs/13520670168/job/37779048967#step:8:352

Reply:
I think this is a bug that the importer shouldn't fail when something different from a XML is given to import. Just don't return early and call super for following the chain.

Reply:
Changed to apply proper fix (IMO).